### PR TITLE
issue 53 -- automatic 'next event' data for home page

### PR DESCRIPTION
**Pull request by DJ-Eckam**

### CODE REVIEW

File: themes/ranked-choice-voting-for-texas-5dabb2094445ea66232bca7e/layout.html [hunk 1]
```diff
@@ -179,7 +179,9 @@ <h3 class="home-hero-heading">Help bring ranked choice voting to Texas!</h3>
                 <h3>News</h3>
                 <ul class="top-news-list">
                   <li>We're actively recruiting &mdash; <a href="/volunteer_1">volunteer today!</a></li>
-                  <li>Next event: Tuesday, Dec. 7. <a href="/calendar_2">View our events calendar.</a></li>
+                  {% for event in site.upcoming_event_pages limit:1 %}
```
Comment (chrisflesher) [inline themes/ranked-choice-voting-for-texas-5dabb2094445ea66232bca7e/layout.html:182]: What about replacing `for event` with `for page`?

File: themes/ranked-choice-voting-for-texas-5dabb2094445ea66232bca7e/layout.html [hunk 2]
```diff
@@ -179,7 +179,9 @@ <h3 class="home-hero-heading">Help bring ranked choice voting to Texas!</h3>
...
+                    <li>Next event: {{ event.event.local_start_at | date: '%A, %b. %-d' }}{% if event.event.is_multi_day? %} through {{ event.event.local_end_at | date: '%A, %b. %-d }}{% endif %}. <a href="/calendar_2">View our events calendar.</a></li>
```
Comment (chrisflesher) [inline themes/ranked-choice-voting-for-texas-5dabb2094445ea66232bca7e/layout.html:183]: How does the %A, %b, and %d work here?

Comment (chrisflesher) [reply themes/ranked-choice-voting-for-texas-5dabb2094445ea66232bca7e/layout.html:183]: Is the second date missing a single quote at the end?

Comment (chrisflesher) [reply themes/ranked-choice-voting-for-texas-5dabb2094445ea66232bca7e/layout.html:183]: Is the `<a href...` missing a closing >?

**Review by chrisflesher — COMMENTED** [review]
I checked and saw this is currently working on the preview but may have some corner cases to fix.

File: themes/ranked-choice-voting-for-texas-5dabb2094445ea66232bca7e/layout.html (hunk 1 above)
Comment (DJ-Eckam) [reply themes/ranked-choice-voting-for-texas-5dabb2094445ea66232bca7e/layout.html:182]: good idea, will do

**Review by chrisflesher — APPROVED** [review]
Looks good!

**Pull request merged**